src/components/Sum.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const Sum = ({ a, b }) => (
```

Comment:
will it work like this?
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```